packages/@webex/plugin-cc/src/services/index.ts

```diff
@@ -1,18 +1,20 @@
 import routingAgent from './agent';
 import {AqmReqs} from './core/aqm-reqs';
+import {WebSocketManager} from './core/WebSocket/WebSocketManager';
 
 export default class Services {
   public readonly agent: ReturnType<typeof routingAgent>;
   private static instance: Services;
 
-  constructor() {
-    const aqmReq = new AqmReqs();
+  constructor(webSocketManager: WebSocketManager) {
```

Comment:
We use DI to inject the websocketmanager from the cc.ts into the sqm-reqs. Preferred this approach over using a singleton system.

Reply:
Non-blocking Q: What is the consensus behind preferring this approach over singleton?

Would we have another entity in the plugin that would reuse this services with a different webSocketManager?